plugin.rb

```diff
@@ -274,6 +274,17 @@ def execute(args)
     end
   end
 
+  DiscourseEvent.on(:post_edited) do |post, topic_changed|
+    if topic_changed && SiteSetting.voting_enabled && UserCustomField.where(value: post.topic_id).where("name = 'votes_archive' OR name = 'votes'").exists?
```

Comment:
`.where("value = ? and name in ('votes', 'votes_archive')", post.topic_id)` 

we should do a follow up PR to introduce a const for "votes" and "votes_archive" cause it is somewhat not nice